### ISSUE TITLE
Pano log

### ISSUE THREAD

**Issue by laurb9** [laurb9/pano-controller#48]
Record information for each panoramas taken in EEPROM.
Dump the EEPROM over serial when connected to PC, for identifying the photo sets and settings easier.

**Comment by Defragster** [comment]
The $8 PJRC prop shield has 8MB Flash and a HID software transfer app.

But an ESP8266 could save them to onboard 3MB SPIFFS and also post them in real time to the control web page to cut and paste to a file - or save the file for FTP transfer when done.

Recording the camera IMAGE number will make it easier to associate the notes - even if using a fresh flash card for each set.

**Comment by laurb9** [comment]
Yes, recording more detail such as image numbers with timestamps for each photo is a good idea, thanks. Maybe elevation and azimuth also, if I add a compass could help with prealignment.

These aren't needed in real time though, since all the stitching happens at home base anyway.

**Comment by laurb9** [comment]
May not need to be in EEPROM at all if it can be transferred to the app immediately.
Logging the position, timestamps and other known metadata for each photo is still useful.
